src/pages/App.js

```diff
@@ -16,6 +17,7 @@ function App() {
       </header>
       <div className={styles.app}>
         <MainVisualization coverageData={coverageData} className={styles.app} />
+        <PastFiles />
```

Comment:
Does this need to be added to the main app? We can see the PastFiles component on the FileUpload page, which you should be able to see when you visit `/file-upload`

Reply:
I did not see that we already had that route! Removed in commit [665b9b7](https://github.com/mcode/mcode-coverage-checker/pull/20/commits/665b9b7e798824bb66e55b199805a8995958adf0).